selfapi.js

```diff
@@ -479,15 +506,24 @@ API.prototype = {
         var example = handler.examples[0];
 
         var request = example.request || {};
-        if (request.headers || request.body) {
+        if (request.headers || request.body || request.urlParameters) {
           markdown += '### Example input:\n\n';
+
+          // Format example-specific URL
+          var exampleURL = '    ' + method.toUpperCase() + ' ' + fullPath;
+          var requestParameters = request.urlParameters || {};
+          for (var parameter in requestParameters) {
+            exampleURL = exampleURL.replace(':' + parameter, requestParameters[parameter]);
+          }
+          markdown += exampleURL + '\n';
```

Comment:
Same suggestion as above, maybe only add an example URL if it's not exactly the same as the header URL above?